.github/workflows/docker-image-perception-master.yml

```diff
@@ -3,6 +3,8 @@ name: Docker Image CI - Master Branch
 on:
   push:
     branches: [master]
+  schedule:
+    - cron: "12 0 * * *"
```

Comment:
Don't add schedule on master for now, it will fail because of package version, we have to find a fix for that.